src/sign-in-page.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-unused-vars */
+import React, { Component } from 'react';
+import './sign-in-page.css';
+// import { template } from 'handlebars';
+/* eslint-enable no-unused-vars */
+
+class SignInPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            userNameEmail: ""
+        };
+
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleInputChange = this.handleInputChange.bind(this);
+    }
+
+    handleSubmit(event) {
+        event.preventDefault();
+        console.log('here');
```

Comment:
Were you using this for debugging? Since it's not outputting anything informative to the console I think we should remove it.

Reply:
yea that is for debugging, I forgot to remove it